config/environments/test.rb

```diff
@@ -41,11 +41,9 @@
   # config.action_view.raise_on_missing_translations = true
 
   # paperclip file store location
-  Paperclip::Attachment.default_options[:path] = "#{Rails.root}/spec/test_files/:class/:id_partition/:style.:extension"
+  Paperclip::Attachment.default_options[:path] = ':rails_root/public:url'
+  Paperclip::Attachment.default_options[:url]  = '/test_files/:class/:id_partition/:style/:filename'
```

Comment:
Why do you think this is better?
I can't find anyone messing with their `:url` option of Paperclip for testing purposes (see [here](https://github.com/search?utf8=%E2%9C%93&q=Paperclip%3A%3AAttachment.default_options%5B%3Aurl%5D+filename%3Atest.rb&type=Code&ref=searchresults))